packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -57,21 +57,25 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
           borderRadius: radioInnerCircleRadius,
           height: radioInnerCircleSize,
           width: radioInnerCircleSize,
+          backgroundColor: tokens.radioFill,
           left: 4,
           top: 4,
         },
       }),
       ['radioVisibility'],
     ),
-    label: buildProps((tokens: RadioTokens) => ({
-      variant: tokens.variant,
-      style: {
-        marginTop: 2,
-        borderStyle: 'solid',
-        borderColor: tokens.textBorderColor,
-        borderWidth: 2,
-        color: tokens.labelColor,
-      },
-    })),
+    label: buildProps(
+      (tokens: RadioTokens) => ({
+        variant: tokens.variant,
+        style: {
+          marginTop: 2,
+          borderStyle: 'solid',
+          borderColor: tokens.textBorderColor,
+          borderWidth: 2,
```

Comment:
Should borderWidth be in Tokens? Because currently borderWidth is not customizable

Reply:
Thanks for pointing that out! Added to tokens. It wasn't highlighted in the web spec, so I didn't realize it was customizable.